nsv13/code/game/gamemodes/overmap/objectives/board_ship.dm

```diff
@@ -19,7 +19,7 @@
 	target_ship.name = ship_name
 	// give it a home
 	var/list/candidates = list()
-	for(var/datum/star_system/random/S in SSstar_system.systems)
+	for(var/datum/star_system/S in SSstar_system.neutral_zone_systems)
 		if(!S.hidden)
 			candidates += S
```

Comment:
This will complicate things if we need to unmerge the new starmap for any reason since the syndicate don't own any neutral zone systems in the old version.

Reply:
If I'm assuming we won't go back, this method should be default and the old method should be fallback, right?